AncientReligions/common/landed_titles/ACR_holy_sites.txt

```diff
@@ -1,19 +1,24 @@
-### Hellenic Pagan
-#
-#b_lebedos = {
-#	# Temple of Artemis at Ephesus
-#	holy_site = hellenic_pagan
-#}
-#b_lampsakos = {
-#	remove_holy_site = hellenic_pagan
-#}
-#b_piraeaus = {
-#	remove_holy_site = hellenic_pagan
-#}
-#b_pyrgos = {
-#	# Statue of Zeus at Olympia
-#	holy_site = hellenic_pagan
-#}
+### Hellenic Pagan Heresies (same as vanilla Hellenic)
```

Comment:
Good point, I'll cut it and see if it keeps working.
When I improve the flavor for these heresies a little I'd like to have a couple of the holy sites differ.